### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by andyren12**

### CODE REVIEW

**Review by elizabethkim11 — COMMENTED** [review]
lgtm! :D

File: styles/BoardCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 450px;
```
Comment (nendow02) [inline styles/BoardCard.module.scss:5]: Can we change the dimensions of the cards when the page becomes smaller than a certain size. My main issue is when only one column can fit, leaving a lot of white space like below. `@media` css property should be able to help you with this change.
image

**Review by nendow02 — CHANGES_REQUESTED** [review]
Looks great! Only one small change, lmk if you have different ideas on how to approach this too.

**Comment by ArshMalik02** [comment]
Looks good! Merging.

**Pull request merged**